fe/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -104,6 +104,15 @@
 
 public class Coordinator {
     private static final Logger LOG = LogManager.getLogger(Coordinator.class);
+
+    // The reason of cancellation of this query.
+    // This is used for telling BE whether it need to report query status when being cancelled.
+    public enum CancelReason {
```

Comment:
Because you have defined in protobuf, we should use it to avoid define one thing in different places.

Reply:
ok